packages/contracts/src/healthcheck/HealthChecker.sol

```diff
@@ -9,36 +9,46 @@ import {SafeInitializable} from "../upgradeable/SafeInitializable.sol";
 error HealthCheckFailed();
 
 abstract contract HealthChecker is SafeInitializable, OwnableUpgradeable {
-    event HealthCheckChanged(address);
-    event HealthCheckEnabledChanged(bool);
+    event HealthCheckChanged(address healthCheck);
+    event ShutdownLossRatioChanged(uint256 ratio);
+    event HealthCheckEnabledChanged(bool enabled);
+    event HealthCheckTriggered(uint8 result);
+
+    // represents 100%
+    uint256 public constant MAX_BPS = 10_000;
```

Comment:
Are constants not create memory shifts?

Reply:
Nope, id doesn't reserve a memory slot. https://docs.soliditylang.org/en/latest/contracts.html#constants